core/src/main/java/org/infinispan/persistence/manager/OrderedUpdatesManagerImpl.java

```diff
@@ -0,0 +1,160 @@
+package org.infinispan.persistence.manager;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import org.infinispan.commons.util.ByRef;
+import org.infinispan.container.DataContainer;
+import org.infinispan.container.versioning.EntryVersion;
+import org.infinispan.container.versioning.InequalVersionComparisonResult;
+import org.infinispan.distribution.DistributionInfo;
+import org.infinispan.distribution.DistributionManager;
+import org.infinispan.factories.annotations.Inject;
+import org.infinispan.metadata.Metadata;
+import org.infinispan.util.concurrent.CompletableFutures;
+import org.infinispan.util.logging.Log;
+import org.infinispan.util.logging.LogFactory;
+
+public class OrderedUpdatesManagerImpl implements OrderedUpdatesManager {
+   private static final Log log = LogFactory.getLog(OrderedUpdatesManagerImpl.class);
+
+   private DataContainer dataContainer;
```

Comment:
don't you wanna put the type? `private DataContainer<Object, Object> dataContainer`